src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -9,12 +9,30 @@
 from marqo.errors import InternalError
 from pydantic import BaseModel
 from marqo.tensor_search.utils import get_best_available_device
+from typing import List, Dict
 
 
 class AddDocsParamsConfig:
     arbitrary_types_allowed = True
 
 
+class AddDocsBodyParamsOld(BaseModel):
+    __root__: List[Union[dict, Any]]
+
+
+class AddDocsBodyParamsNew(BaseModel):
```

Comment:
I'd call this AddDocsBodyParams because once the old code is removed the name will be confusing

Reply:
Updated